content/en/tracing/profiler/connect_traces_and_profiles.md

```diff
@@ -86,6 +86,14 @@ To enable Code Hotspots identification for Go, [turn on profiling for your servi
 [10]: https://go-review.googlesource.com/c/go/+/369741/
 [11]: https://go-review.googlesource.com/c/go/+/369983/
 {{< /programming-lang >}}
+{{< programming-lang lang=".NET" >}}
+
+Code Hotspots identification is enabled by default when you [turn on profiling for your service][1].
+
+Requires tracing library version 2.7.0 or greater.
```

Comment:
Should we specify Profiling Beta 2 instead or in complement?

Reply:
Code Hotspots is available since 2.7 (i.e. before Beta 2).
I followed the Ruby text.